scripts/tests/twister/test_twister.py

```diff
@@ -77,3 +78,12 @@ def test_testsuite_config_files():
 
     # Check extra kconfig statements, too
     assert scenario["extra_configs"] == ["CONFIG_FOO=y"]
+
+def test_configuration_error():
+    """Test to validate that the ConfigurationError is raisable without further errors.
+
+    Previously, ConfigurationError exhibited additional errors
```

Comment:
Please describe here what the new test does.  This text looks like a commit message.